schema/mysql/upgrades/0.2.0-source-username.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE source ADD COLUMN listener_username varchar(255);
```

Comment:
You could add `AFTER name;` here even there's no equivalent statement for postgres.

Reply:
Forgot we were doing this for MySQL. Changed, thanks.